doc/changes/names.rst

```diff
@@ -3,7 +3,7 @@
     If you want to add to add yourself to the list of authors,
     please edit CITATION.cff and run maint_tools/citation_cff_maint.py.
 
-
+    
```

Comment:
Looks spurious to me ?

Reply:
yup: I will fix the script that generates this.

what must have happened:
- the script generates a file with some spaces at the end of the line
- the file gets committed with pre-commit that removes the spaces
- when CI eventually runs, it regenerates the files and commits just the spaces